webtransport/README.md

```diff
@@ -0,0 +1,52 @@
+# libp2p WebTransport
+
+| Lifecycle Stage | Maturity                 | Status | Latest Revision |
+|-----------------|--------------------------|--------|-----------------|
+| 1A              | DRAFT                    | Active | r0, 2022-04-03  |
+
+Authors: [@marten-seemann]
+
+Interest Group: TODO
```

Comment:
```suggestion
Interest Group: [@mxinden]
```

Happy to be listed here.

Reply:
I also added Marco, given that he's writing the JS implementation.